internal/svcctl/svcctl.go

```diff
@@ -121,22 +121,26 @@ func startAndWait(ctx context.Context, ipComm IPCommunicator, exec string) error
 }
 
 func waitUp(ctx context.Context, ipComm IPCommunicator) error {
-	for try := 1; try <= 24; try++ {
+	start := time.Now()
+	for try := 1; try <= 32; try++ {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 		default:
 		}
 
-		start := time.Now()
+		tryStart := time.Now()
 		timeout := time.Millisecond * time.Duration(try*try)
 
-		logging.Debug("Attempt: %d, timeout: %v", try, timeout)
+		logging.Debug("Attempt: %d, timeout: %v, total: %v", try, timeout, time.Since(start))
 		if err := ping(ctx, ipComm, timeout); err != nil {
-			if !errors.Is(err, errNotUp) {
+			if errors.Is(err, ctlErrNotSureIfUp) {
+				continue
```

Comment:
Could you add a comment explaining why there's no delay in retrying when this error is encountered?